### DISCUSSION TITLE
make the quater helixes for the multidimention

### DISCUSSION THREAD

**Discussion by TheClaudeQC** [OpenRCT2/OpenRCT2#20648] [Feature requests]
I did try the quater helixes whit a multidimention (the flying coaster is the best to build it in the first place), then come back as a multidimention coaster, I then putted the Compact inverted coaster track (as it have the same sprites as the multidimention when it is inverted). and it looked good, but it could be great if the multidimention have it and maybe get the compact inverted sprites and make it a bit lower compared to the terrain.

**Comment by astraylife1** [comment]
I wish we had those quarter helix pieces on a lot of the coasters. Especially the unbanked quarter helix for the wooden coaster. Would be awesome for trains kea big the station to have a gentle sloped turn to the lift. 

But I never even thought about your fox with compact invert matching the multi. Seems like it could be feasible if it looks good
